app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<p><%= @recipe.name %></p>
+<% @recipe.ingredients.each do |ingredient| %>
+  <p><%= ingredient.name %></p>
+<% end %>
+<p>Total Cost: <%= @recipe.total_cost %></p>
+
+<%= form_with url: "/recipes/#{@recipe.id}", method: :patch, data: {turbo: false}, local: true do |form| %>
```

Comment:
This is where you might want to make the path and routing more RESTful, something like 
`POST /recipes/:recipe_id/ingredients` to `recipe_ingredients#create`